person.rb

```diff
@@ -2,27 +2,31 @@
 require './rental'
 
 class Person
-  attr_accessor :name, :age, :rentals
-  attr_reader :id
-
   def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] For some reason, when I try to create a new teacher, this line returns the following error:
![Captura de pantalla 2022-04-08 153116](https://user-images.githubusercontent.com/59100477/162445971-6f704fc1-9d0d-4a37-a3c1-e5fb11c5398c.jpg)

Saying that the `Person` class has been wrongly initialized because it's not expecting to receive 3 arguments, but it expects to receive 1 or 2 only. All other features are working properly, so it may be a typo or a missing coma probably, please take care of this issue as this is the only problem on your app right now!

Reply:
Fixed, thanks